README.md

```diff
@@ -114,6 +114,15 @@ volume:
   type: ROOT
   size: 64
 ```
+===============
+### Testing
+
+```bash
+$ virtualenv -p python3 venv
+$ source venv/bin/activate
+$ pip install -r testing/requirements.txt
+$ py.test frameworks/helloworld/tests
```

Comment:
`test.sh` includes wrapper features that aren't relevant to developers (CCM, jenkins, etc.).  `py.test` is a standard that people are familiar with.  We shouldn't introduce a non-standard tool unless it provides valuable features.